src/sockets/commands/admin/areset.ts

```diff
@@ -0,0 +1,42 @@
+import { Command } from '../types';
+import { SocketUser } from '../../types';
+import User from '../../../models/user';
+import seasonNumber from '../../../util/seasonNumber';
+
+export const areset: Command = {
+  command: 'reset',
+  help: "/reset all plyers's rank data and start a new season",
+  run: async (args: string[], socket: SocketUser) => {
+    try {
+      const users = await User.find({});
```

Comment:
Let's break this function up to a couple different steps. There's going to be some complex logic we'll want to test which we can't directly through this `run()` method. 

Let's sit down sometime and discuss how to structure this.

Reply:
Actually, I do not fully understand it, just follow the same command to do it😂

Reply:
Yep, remind me in our next meeting to chat about this :)